cockroachcloud/cluster-overview-page.md

```diff
@@ -20,17 +20,17 @@ The **Cluster settings** panel displays the settings you chose during [cluster c
 
 | Field     | Description                                                                                             |
 |-----------|---------------------------------------------------------------------------------------------------------|
-| Cloud     | The cluster's [cloud provider](create-a-serverless-cluster.html#step-2-select-a-cloud-provider-region). |
+| Cloud     | The cluster's [cloud provider](create-a-serverless-cluster.html#step-2-select-the-cloud-provider). |
 | Plan type | The plan type used to create the cluster.                                                               |
-| Region    | The cluster's [region](create-a-serverless-cluster.html#step-2-select-a-cloud-provider-region).         |
+| Region(s)    | The cluster's [region(s)](create-a-serverless-cluster.html#step-3-select-the-region-s).         |
```

Comment:
Do we need the (s)? What about just "Regions"?

Reply:
was worried about implying that you have to have more than one region, but maybe that was overthinking

Reply:
What about "at minimum one and at most six" where you currently talk about the maximum? I'd think it would be obvious that you can't have less than 1, though.